.github/workflows/ci.yaml

```diff
@@ -288,8 +288,11 @@ jobs:
       - name: Build standalone release
         run: source scl_source enable devtoolset-9 && yarn release:standalone
 
-      - name: Sanity test standalone release
-        run: yarn test:standalone-release
+      - name: Install test dependencies
+        run: yarn install --ignore-scripts && cd test && yarn install
+
+      - name: Run integration tests on standalone release
+        run: yarn test:integration
```

Comment:
NOTE: I had to use `--ignore-scripts` because otherwise `yarn install` fails if you don't checkout the repo with submodules. On the Linux job that runs on CentOS7, it didn't have the latest `git` so this one-liner was the best alternative. It ensures we have the right TS/Jest dependencies to run the `yarn test:integration` step.